examples/pagedir/.env

```diff
@@ -0,0 +1 @@
+UPLOADTHING_URL="http://localhost:3010"
```

Comment:
Ah, this is why you removed `.env` from the gitignore. 

I think this should probably be an `example.env`, since you don't actually want this most of the time, no?

Reply:
Sure